server/services/alert_service/aws_sns/text_messaging.go

```diff
@@ -0,0 +1,52 @@
+package awssns
+
+import (
+	"main/server/request"
+	"main/server/response"
+	"main/server/utils"
+	"os"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/sns"
+	"github.com/gin-gonic/gin"
+)
+
+func SendSMS(ctx *gin.Context, reqBody request.AwsTextMessagingRequest) {
+
+	// Create Session and assign AccessKeyID and SecretAccessKey
+	sess, err := session.NewSession(&aws.Config{
+		Region:      aws.String(os.Getenv("AWS_REGION")),
+		Credentials: credentials.NewStaticCredentials(os.Getenv("AWS_ACESSS_KEY_ID"), os.Getenv("AWS_SECRET_ACCESS_KEY"), ""),
+	},
+	)
+	if err != nil {
+		response.ShowResponse(err.Error(), utils.HTTP_INTERNAL_SERVER_ERROR, utils.FAILURE, nil, ctx)
+		return
+	}
+
+	// Create SNS service
+	svc := sns.New(sess)
```

Comment:
what is svc?

Reply:
svc is the instance of sns with a session.

Reply:
What does `svc` variable name stand for?

Reply:
Sir,It stands for service. It is common name for variable when using aws services.